README.md

```diff
@@ -77,8 +77,8 @@ Generally, it is recommended to use the latest release of kube-state-metrics. If
 | **v2.7.0**         | v1.25                        |
 | **v2.8.2**         | v1.26                        |
 | **v2.9.2**         | v1.26                        |
-| **v2.10.0**        | v1.27                        |
-| **main**           | v1.27                        |
+| **v2.10.1**        | v1.27                        |
```

Comment:
(FYI) `main` needs to be updated to reflect the latest `2.10.x` version in the `README.md` so as to not cause conflicts when merging `2.10` back into `main`.